tests/main_Rog_N_loop.jl

```diff
@@ -0,0 +1,64 @@
+using ITensors
+using Plots
+using Measures
+include("../src/evolution.jl")
+include("../src/constants.jl")
+
+# This file evolves the system under the vaccum oscillations + self-interaction
+# Hamiltonian and then plots survival probability for 
+# multiple system sizes through loops. 
+
+function main(N)
+    cutoff = 1E-14
+    τ = 0.05
+    ttotal = 5
+    tolerance  = 5E-1
+    Δx = 1E-3
+    s = siteinds("S=1/2", N; conserve_qns=false)
+    a_t = 0.965
+    b_t = 0
+    c_t = 0
+    mu = ones(N)
+    n = mu .* fill((Δx)^3/(sqrt(2) * G_F), N)
+    B = [0, 0, -1]
+    ω_a = fill(2, div(N, 2))
+    ω_b = fill(0, div(N, 2))
+    Δω = (ω_a - ω_b)/2
+    ω = vcat(ω_a, ω_b)
+    ψ = productMPS(s, n -> n <= N/2 ? "Dn" : "Up")
+    Sz_array, prob_surv_array = evolve(s, τ, n, ω, B, N, Δx, ψ, cutoff, ttotal)
+    function find_first_local_minima_index(arr)
```

Comment:
I require the "first" minimum from the arrays since Rogerro plots that too. I can't find how the relative minimum will be the same as the first minimum in scipy.signal.argrelextrema for all sites?

Reply:
I think that function returns a list of minima, so you have to choose the first one. But, no action required for now, of course.